src/app/modules/documentation-module/common/DocLanding/index.tsx

```diff
@@ -1,15 +1,19 @@
-//cc:api documentation module#; content - landing
+// cc:api documentation module#; content - landing
 import React from 'react';
-import { Box, Grid, Typography } from '@material-ui/core';
+import { Box, CircularProgress, Grid, Typography } from '@material-ui/core';
```

Comment:
unused import right? this is moved to loadingindicator.tsx?

Reply:
yep!